recipes/hiveplot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.1.8.4" %}
+{% set pkgname = "hiveplot" %}
+{% set maintainer = "ericmjl" %}
```

Comment:
Let's please drop this. It is really better to have a proper listing in the maintainers' section.